src/plotrecipes/data.jl

```diff
@@ -2,7 +2,7 @@
 # Licensed under the ISC License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
-@recipe function f(sdata::AbstractData; variables=nothing)
+@recipe function f(sdata::AbstractData, variables=nothing)
```

Comment:
Nice, I would even rename the argument to `vars` to avoid conflicting with the `GeoStatsBase.variables` function. :)

Reply:
Oh, you made that change recently, I almost missed it. Thanks :smile: